jni/themis_jni.mk

```
@@ -81,7 +81,7 @@ endif
 	         } \
 	       }' \
 	 ) && \
-	 jnidir=$$(cd "$(jnidir)" && pwd) && \
+	 jnidir=$$(cd "$(DESTDIR)$(jnidir)" && pwd) && \
```

As a quick&dirty solution we can probably do
```suggestion
	 jnidir=$$(cd "$(jnidir)" && pwd || echo "$(jnidir)") && \
```
but there should be better way.

Linuxes typically have `realpath` but it's not available on BSDs like macOS.

The most ‘portable’ approach I've found on the Internet is use Python:

```
python -c "import os,sys; print(os.path.realpath(sys.argv[1]))" $(jnidir)
```

but, well, it requires Python :)

---

Or we could just do

```suggestion
	 jnidir=$$(echo "$(jnidir)" | sed -E 's=/{2,}=/=g') && \
```

and call it a day.